### PR TITLE
Send information about external graphic to the print server

### PR DESCRIPTION
**Pull request by rmichaelis**

### CODE REVIEW

**Comment by rmichaelis** [comment]
I'm done with the modification. 
Could you please review ?

File: src/services/print.js [hunk 1]
```diff
@@ -434,13 +425,37 @@ ngeo.Print.prototype.encodeVectorLayer_ = function(arr, layer, resolution) {
...
+          styledFeature.setGeometry(geometry);
+          geojsonFeature = geojsonFormat.writeFeatureObject(styledFeature);
+          geometry = styledFeature.getGeometry();
+          styledFeature = null;
+          geojsonFeatures.push(geojsonFeature);
+        } else {
+          geojsonFeature = origGeojsonFeature;
+          geometry = originalFeature.getGeometry();
+          // no need to encode features with no geometry
+          if (!goog.isDefAndNotNull(geometry)) {
```
Comment (fredj) [inline src/services/print.js]: can be simplifed to `if (!geometry) {`

**Comment by fredj** [comment]
LGTM, @sbrunner do you want to take a look?

**Comment by sbrunner** [comment]
Looks good for me to :-)

**Comment by rmichaelis** [comment]
```!geometry``` instead of  ```geometry !== null```  breaks the code.

**Comment by rmichaelis** [comment]
Thanks both of you for the review. 
Modified as suggested.

**Comment by fredj** [comment]
Thanks @rmichaelis

**Pull request merged**